### PR TITLE
 CLA - Add upload from remote server

### PR DESCRIPTION
**Pull request by alesyalis**
CLA added the ability to download images via url

### CODE REVIEW

**Comment by alesyalis** [comment]
I have read the CLA Document and I hereby sign the CLA

File: Exadel.Compreface/Helpers/FileHelpers.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+namespace Exadel.Compreface.Helpers;
+
+public class FileHelpers
+{
+    public static string GenerateFileName(string filePath)
+    {
+        if (File.Exists(filePath))
+        {
+            throw new FileNotFoundException(message: $"file does not exist in path : {filePath}!!!");
```
Comment (Evgeniy-Sinicin) [inline Exadel.Compreface/Helpers/FileHelpers.cs:9]: If file exists, file doesn't exist? 😅🤯

Comment (alesyalis) [reply Exadel.Compreface/Helpers/FileHelpers.cs:9]: misprint

**Review by Ukrainiangirl — APPROVED** [review]

**Comment by Ukrainiangirl** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by alesyalis** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by Ukrainiangirl** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by Evgeniy-Sinicin — APPROVED** [review]

**Comment by alesyalis** [comment]
recheckcla

**Pull request closed without merging**